### PR TITLE
Build instructions

### PR DESCRIPTION
**Pull request by realaravinth**
Instructions for the following modes of deployment have been added:
- bare metal setup
- docker 
- docker-compose

Please merge it after #35

### CODE REVIEW

File: docs/DEPLOYMENT.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+For Debian based systems:
+
+```bash
+sudo apt install postgresql
+```
+
+### 2. Create new user for running `Triox`:
+
+```bash
+ sudo useradd -b /srv -m -s /usr/bin/zsh triox
```
Comment (AaronErhardt) [inline docs/DEPLOYMENT.md:64]: zsh is nice but we can assume someone has installed it. For this purpose bash is probably better

File: docs/DEPLOYMENT.md [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+
+```bash
+ sudo -iu postgres # switch to `postgres` user
+ psql
+```
+
+Postgres shell:
+
+```psql
+ postgres=#  CREATE USER triox WITH PASSWORD 'my super long password and yes you need single quote`;
```
Comment (AaronErhardt) [inline docs/DEPLOYMENT.md:77]: Is is intentional that the closing quote is a ` and not a ' ?

**Review by AaronErhardt — CHANGES_REQUESTED** [review]
Well done!

File: docs/DEPLOYMENT.md (hunk 2 above)
Comment (realaravinth) [reply docs/DEPLOYMENT.md:77]: no, I make lots of typos

**Pull request merged**